examples/ftp/Ftp.cpp

```diff
@@ -190,12 +190,11 @@ int main()
                 break;
             }
         }
-
-    } while (choice != 0);
+    }
+    while (choice != 0);
```

Comment:
I guess there's no option for do-while loops? I can live with that change, since it's not used that much, but find the other formatting better.

Reply:
That's totally possible. I thought we wanted the brace before the while but I'll remove that instead.